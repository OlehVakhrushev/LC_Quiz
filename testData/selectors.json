{
  "userMenu": ".ant-dropdown-trigger.ant-dropdown-link",
  "menuItem": ".ant-dropdown-menu-item-only-child"
}
