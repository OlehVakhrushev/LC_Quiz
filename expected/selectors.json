{
  "alertNameTryAgain": ".alert-heading",
  "alertT": "//p",
  "alertRecommend": ".mb-0",
  "textSecondary": ".text-secondary",
  "rightAnswer": ".mr-4",
  "resetResult": ".ant-btn.ant-btn-default",
  "section2": ".edit.ml-2",
  "quizLink": ".mr-2"
}


